mantidproject/mslice
# Fix grouping workspaces

The Group workspaces was not working. This pull requests fixes it

**To test:**
1.Load multiple workspaces into mslice
2. Select them all via using `ctrl` + `click`
3. Click on the `Group Selected` button in the **Workspace Manager**
4. Verify that a new group has appeared.

Fixes #85
